libobs-opengl/gl-cocoa.m

```diff
@@ -482,15 +482,13 @@ void gl_getclientsize(const struct gs_swap_chain *swap, uint32_t *width, uint32_
 
             break;
         }
+        case 0: /* case 0 is workaround for Syphon always providing invalid IOSurfaces */
```

Comment:
Could be more specific that this is before https://github.com/Syphon/Syphon-Framework/commit/71351d4b484cd2d1917867f7846a5cdca724552d or https://github.com/Syphon/Syphon-Framework/commit/2dc6d319514e698900b7af6dd9a750b66700137a, but it might not matter much.